src/terraform/install.sh

```diff
@@ -180,23 +180,29 @@ check_packages() {
 # Function to fetch the version released prior to the latest version
 get_previous_version() {
     REPO_URL=$1
-    curl -s "${REPO_URL}/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/'
+    if command -v jq &> /dev/null; then
+        curl -s "${REPO_URL}/latest" | jq -r '.tag_name'
+    else 
+        curl -s "${REPO_URL}/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/'
+    fi
 }
 
```

Comment:
Let's pause on this PR for an additional fallback logic until https://github.com/devcontainers/features/pull/914 is merged in. Then, you can follow the same logic here!

Reply:
sure !